app/views/handlers/question.py

```diff
@@ -128,6 +128,9 @@ def get_context(self):
                 )
             )
 
+        if self.form.errors or self.form.question_errors:
+            self.page_title = "Error: " + self.page_title
```

Comment:
- This needs to be translatable, so should be wrapped in a call to `gettext` (this is done [here](https://github.com/ONSdigital/eq-questionnaire-runner/blob/master/app/forms/field_handlers/dropdown_handler.py#L12))
- We should use a placeholder for the page title so that a translator has more context

```suggestion
            self.page_title = gettext("Error: {page_title}").format(page_title=self.page_title)
```